om-2.0.rdf

```diff
@@ -21442,6 +21479,152 @@
+              <om:SingularUnit rdf:about="&om;voltampere"/>
+              <om:SingularUnit rdf:about="&om;kilovoltampere"/>
+              <om:SingularUnit rdf:about="&om;megavoltampere"/>
+            </owl:oneOf>
+          </owl:Class>
+        </owl:unionOf>
+      </owl:Class>
+    </owl:equivalentClass>
+  </owl:Class>  	  
+
+  <owl:Class rdf:about="&om;ApparentPower">
+    <rdfs:subClassOf>
+      <owl:Restriction>
+        <owl:onProperty rdf:resource="&om;hasValue"/>
+        <owl:allValuesFrom>
+          <owl:Restriction>
+            <owl:onProperty rdf:resource="&om;hasUnit"/>
+            <owl:allValuesFrom>
+              <owl:Class rdf:about="&om;ApparentPowerUnit"/>
+            </owl:allValuesFrom>
+          </owl:Restriction>	
+        </owl:allValuesFrom>
+      </owl:Restriction>     
+    </rdfs:subClassOf>     
+  </owl:Class>
+
+  <!-- Volt-Ampere Reactive Ontology -->
+  <om:Unit rdf:about="&om;voltamperereactive">
+    <rdfs:label xml:lang="en">volt-ampere reactive</rdfs:label>
+    <rdfs:label xml:lang="nl">volt-ampere reactive</rdfs:label>
+    <rdfs:comment xml:lang="en">Volt-Ampere Reactive (VAR) is a unit of reactive power in a power system. Reactive power is the power associated with the magnetic and electric fields that are produced in a circuit when voltage and current are out of phase.</rdfs:comment>
+    <om:longcomment>Reactive power is an important parameter in power system analysis, as it affects the voltage regulation and power factor of the system. Electrical equipment such as motors, transformers, and capacitors can produce or consume reactive power, and the proper management of reactive power is important for maintaining the stability and efficiency of the power system.</om:longcomment>
+    <om:symbol>VAR</om:symbol>
+    <rdf:type rdf:resource="&om;SingularUnit"/>
+    <om:hasDimension rdf:resource="&om;power-Dimension"/>
+  </om:Unit>
+
+  <!-- Volt-Ampere Reactive Multiples and Submultiples Ontology -->
+
+  <om:PrefixedUnit rdf:about="&om;kilovoltamperereactive">
+    <rdfs:label xml:lang="en">kilovolt-ampere reactive</rdfs:label>
+    <rdfs:label xml:lang="nl">kilovolt-ampere reactive</rdfs:label>
+    <rdfs:comment xml:lang="en">The kilovolt-ampere reactive is a unit of reactive power defined as 1.0e3 volt-ampere reactive.</rdfs:comment>
+    <om:symbol>kVAR</om:symbol>
+    <om:hasPrefix rdf:resource="&om;kilo"/>
+    <!-- hasSingularUnit --> <om:hasUnit rdf:resource="&om;voltamperereactive"/>
+    <om:hasDimension rdf:resource="&om;power-Dimension"/>
+  </om:PrefixedUnit>
+
+  <om:PrefixedUnit rdf:about="&om;megavoltamperereactive">
+    <rdfs:label xml:lang="en">megavolt-ampere reactive</rdfs:label>
+    <rdfs:label xml:lang="nl">megavolt-ampere reactive</rdfs:label>
+    <rdfs:comment xml:lang="en">The megavolt-ampere reactive is a unit of electric charge defined as 1.0e6 volt-ampere reactive.</rdfs:comment>
+    <om:symbol>mVAR</om:symbol>

```

Comment:
Same as above: "m" is milli, "M" is mega. Please correct.

Reply:
Corrected.